src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
```

Comment:
we use it?

Reply:
I'm not sure about the usage of that file, i believe it came with the react project